src/web/src/pages/CourseScheduler.vue

```diff
@@ -457,6 +462,19 @@ export default {
 };
 </script>
 
+<!-- All Styling For Dark Mode For Course Information Modals.  -->
+<style scoped>
+/deep/ .onStyleDarkMode {
+  background: var(--dark-primary);
+  color: white;
```

Comment:
let's use the `dark.scss` variables when we can, so `color: var(--dark-text-primary)` or secondary

Reply:
But I thought var(--dark-primary) is the Color described in the Dark.CSS File?

Reply:
Yeah, Im referring to the `color: white`

Reply:
Oh alright that makes sense. I'll try that out and see if it works. If not, I'll let you all know.

Reply:
It works I believe. Can you all check it out? I think it's ready for Merge into Master.